### ISSUE TITLE
sap/ui/layout/form/SimpleForm: layout property is not checked properly

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#1066]
When defining a deprecated value for the `sap/ui/layout/form/SimpleForm#layout` property in XML View / Fragment, UI5 linter should report a finding. 

```xml
<form:SimpleForm
      id="myForm"
      editable="false"
      layout="GridLayout"
      title="Address"
>
```

The `layout` property is of an enum of type https://ui5.sap.com/#/api/sap.ui.layout.form.SimpleFormLayout. In the example above the `GridLayout` enum value is deprecated.

# Further information
- [sap/ui/layout/form/SimpleForm](https://ui5.sap.com/#/api/sap.ui.layout.form.SimpleForm)
- [sap/ui/layout/form/SimpleFormLayout](https://ui5.sap.com/#/api/sap.ui.layout.form.SimpleFormLayout)

**Comment by flovogt** [comment]
This will be fixed with https://github.com/UI5/linter/pull/621/.

JIRA: CPOUI5FOUNDATION-1089

**Comment by flovogt** [comment]
Implemented with [v1.23.0](https://github.com/UI5/linter/releases/tag/v1.23.0)
